V8: Meaningful exception when the endpoint instance is stoped (#5024)
Particular/docs.particular.net: nservicebus/upgrades/7to8/index.md

## NServiceBus Azure Host

The `NServiceBus.Hosting.Azure` and `NServiceBus.Hosting.Azure.HostProcess` are deprecated.See the [NServiceBus Azure Host upgrade guide](/nservicebus/upgrades/acs-host-7to8.md) for details and alternatives.

> ## Meaningful exceptions when stopped

Reviewer: ```suggestion
## Attempts to access the endpoint instance after stopping now throws
```

> ## Meaningful exceptions when stopped

Reviewer 2: ```suggestion
## Accessing the endpoint after stopping throws an exception
```

> The core is now throwing an `InvalidOperationException` when invoking message opererations on `IMessageSession` when the endpoint instance is stopping or stopped to indicate that the instance can no longer be used.

Reviewer: ```suggestion
Using any of the messaging operations exposed on the endpoint instance will now throw an `InvalidOperationException` when invoking message operations on `IMessageSession` when the endpoint instance is either stopped or in the process of being stopped to prevent indeterministic behaviour.
```
